src/js/visual/configurator-csr.js

```diff
@@ -1373,8 +1377,9 @@ ripe.ConfiguratorCsr.prototype._loadCsrAssets = async function(
     this.dracoLoader = ripe.CsrUtils.loadDracoLoader();
 
     // computes environment file URL
-    const envUrl = this.owner.get3dSceneEnvironmentUrl();
-    const envFormat = "hdr";
+    const environment = sceneOptions.environment;
+    const environmentFormat = "hdr";

```

Comment:
I'd omit this since it's the default option

Reply:
It's not because I needed to define the format, it's because is a hardcoded value because this logic is not defined.

Reply:
`loadEnvironment` defaults to 'hdr' right? https://github.com/ripe-tech/ripe-sdk/blob/563ba2394316f4ce2fdb6ecbed7f818afa246997/src/js/visual/csr/utils.js#L270

`get3dSceneEnvironmentUrl` fallsback to 'hdr' as well

Reply:
This is here to enforce exactly to use `hdr`, it's not because it needs to pass it to `loadEnvironment()`, it's two different contexts